modules/kubernetes-addons/helm-addon/outputs.tf

```diff
@@ -0,0 +1,10 @@
+output "helm_release_addon" {
+  description = "Describes the desired status of a chart in a kubernetes cluster."
+  value = try(
+    {
+      for k, v in helm_release.addon : k => (
```

Comment:
I think the value here would be: 
```hcl
value = try({ for k, v in helm_release.addon : k => v if k != "repository_password" }, {})
```

Reply:
Either works in my testing.
I tend to go with not setting explicitly anything, hence null.
But not all providers handle that well.
I'm fine following your existing style, if you prefer.